modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/cluster/init/ClusterInitOptions.java

```diff
@@ -216,8 +216,8 @@ private DuplicatesChecker(String optionToCheck) {
         }
 
         private boolean hasDuplicate(List<String> args) {
-            var arr = args.toArray(String[]::new);
-            for (var str : arr) {
+            String[] arr = args.toArray(String[]::new);
```

Comment:
Probably not related to your PR, but why do we need this array conversion?

Reply:
Wasn't looking at that, just fixed the var usage violation. Agree, the DuplicatesChecker looks weird. I'd leave it as is, it works and it isn't on any critical path - the first person who needs to really change it will surely refactor it.